graphistry/plotter.py

```diff
@@ -653,3 +653,49 @@ def _make_vgraph_dataset(self, edges, nodes, name):
         dataset = vgraph.create(elist, filtered_nlist, sources, dests, nodeid, node_map, name)
         dataset['encodings'] = encodings
         return dataset
+
+
+    def bolt(self, driver):
+        from neo4j import GraphDatabase, Driver
+        if driver is not None and isinstance(driver, Driver) == False:
+            driver = GraphDatabase.driver(**driver)
+        res = copy.copy(self)
+        res._bolt_driver = driver
+        return res
+
+
+    def cypher(self, query, params={}):
+        res = copy.copy(self)
+        driver = self._bolt_driver or PyGraphistry._config['bolt_driver']
+        with driver.session() as session:
+            graph = session.run(query, **params).graph()
+            edges = Plotter._bolt_graph_to_dataframe(graph)
+            res._edges = edges
+        return res.bind(\
+            edge_title='_bolt_relationship_id',\
+            source='_bolt_source_id',\
+            destination='_bolt_destination_id'
+        )
+
+
+    @staticmethod
+    def _bolt_graph_to_dataframe(graph):
+        import pandas as pd
+        return pd.DataFrame([
+            Plotter.merge_two_dicts(
```

Comment:
FWIW, probably worth testing we can do a 500K node / 2M edge graph this way, because our users will want that. this function call sticks out wrt that.

Reply:
As it stands, this works with the entire silk road data set from `bolt://violet-russel-mills-natalia.graphstory.cloud:7687`.